packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss

```diff
@@ -9,23 +9,23 @@
 @import '../../styles/shared-styles.scss';
 @import './mgt-file-list.theme.scss';
 
-$file-item-background-color--hover: set-var(list__item__background-color--hover, $theme-default, $file-list);
-$file-item-background-color--active: set-var(list__item__background-color--active, $theme-default, $file-list);
-$background-color: set-var(list__background-color, $theme-default, $file-list);
-$file-list-border-radius: var(--file-list-border-radius, 4px);
-$file-list-box-shadow: var(
-  --file-list-box-shadow,
-  0px 3.2px 7.2px rgba(0, 0, 0, 0.132),
-  0px 0.6px 1.8px rgba(0, 0, 0, 0.108)
-);
+$file-list-border-radius: var(--file-list-border-radius, 8px);
+$file-list-box-shadow: var(--file-list-box-shadow, var(--elevation-shadow-card-rest));
 $file-list-border: var(--file-list-border, none);
-$file-item-border-radius: var(--file-item-border-radius, 2px);
-$file-item-margin: var(--file-item-margin, 0 4px 0 4px);
+$file-list-padding: var(--file-list-padding, 0);
+$file-list-margin: var(--file-list-margin, 0);
```

Comment:
is the removal of the left and right margin intentional here?

Reply:
Yes. I'm _piggy-backing_ on the `.file-item` class and setting the same margin in L55 below, using the `mgt-file-item` tokens. It was one way ~easier~ to do it.